style/README.md

```diff
@@ -1,14 +1,22 @@
 Style
 =====
 
-A guide for programming in style.
+Delivering valuable features
+and working software
+are our ultimate goals.
 
+This style guide should be used
+in service of those goals,
+and not as an end in itself.
+
+Your time is valuable.
 Use [Hound] to automatically review your
 GitHub pull requests for
 style guide violations.
 
 [Hound]: https://houndci.com
 
+
```

Comment:
That's easily the best comment anyone could have left on this PR.

Everyone else - just stop yourselves now.  You are not going to do better than Mike.